src/Workspaces/Core/Portable/Execution/ISolutionSynchronizationService.cs

```diff
@@ -22,6 +22,8 @@ internal interface IRemotableDataService : IWorkspaceService
         /// </summary>
         ValueTask<PinnedRemotableDataScope> CreatePinnedRemotableDataScopeAsync(Solution solution, CancellationToken cancellationToken);
 
+        AssetStorages.Storage? TryGetStorage(int scopeId);
```

Comment:
doc.  (does context make it clear what scopeId means?)

Reply:
I don't know what `scopeId` is. It's the same parameter that gets passed to `GetRemotableDataAsync` below.

Reply:
I believe it's the key for a PinnedRemotableDataScope, and used to identify a storage context which is valid for the duration of a single call from devenv→OOP. @tmat to confirm